library/core/src/cell.rs

```diff
@@ -12,7 +12,7 @@
 //!
 //! Shareable mutable containers exist to permit mutability in a controlled manner, even in the
 //! presence of aliasing. [`Cell<T>`], [`RefCell<T>`], and [`OnceCell<T>`] allow doing this in
-//! a single-threaded way—they do not implement [`Sync`]. (If you need to do aliasing and
+//! a single-threaded way-they do not implement [`Sync`]. (If you need to do aliasing and
```

Comment:
This was intentional. A long dash renders differently from a normal dash/minus sign. After your change it looks like `way-they` is a word

Reply:
Tbh I think we should always recommend space-en-space, space-hyphen-hyphen-space, or even space-hyphen-space as opposed to em with no spaces. Not all style guides accept it, but those variants are reasonably common and it's actually understandable in monospace. Or else rustdoc should rewrite `--`/`---` (wouldn't mind that either).

Reply:
Apparently I've thought that for a while since I originally wrote it this way, mildly amused to find myself on the git log for this very specific dash 😆 https://github.com/rust-lang/rust/commit/9b5122976e9eb0f2f0b8def82fbcd972837880e4